### ISSUE TITLE
Videojs.com fails HTML validator because of missing </a> after Dropbox.com link

### ISSUE THREAD

**Issue by OwenEdwards** [videojs/videojs.com#35]
Yeah, I know, this is really persnickety, but I was checking the www.videojs.com site for accessibility issues and noticed this HTML validation error:

https://validator.w3.org/nu/?doc=http%3A%2F%2Fvideojs.com%2F

````
Error: Unclosed element a.
<li><a href="http://dropbox.com" target="_blank"><img src="/img/company-logos/dropbox.png" alt="Dropbox"></li>
````

There's a missing `</a>`, and because of it the rest of the page can't be validated. I'll create a PR to fix this minor issue if someone can explain to me what branch of this repo corresponds to that site?

**Comment by gkatsev** [comment]
Who uses validators anymore, anway? :P
An unclosed anchor tag is a silly mistake, though. I think the website is based off of the [5.0 branch](https://github.com/videojs/videojs.com/tree/5.0)

**Comment by OwenEdwards** [comment]
Fixed by #40.
